detection_rules/etc/version.lock.json

```diff
@@ -334,11 +334,20 @@
     "version": 2
   },
   "07b5f85a-240f-11ed-b3d9-f661ea17fbce": {
-    "min_stack_version": "8.3",
+    "min_stack_version": "8.4",
+    "previous": {
+      "8.3": {
+        "max_allowable_version": 103,
+        "rule_name": "Google Drive Ownership Transferred via Google Workspace",
+        "sha256": "1c82ea9b65fada4ec684045bd8b3e5eaa0730b35b41ddef3dd151ff26a9d6be9",
+        "type": "query",
+        "version": 4
+      }
+    },
     "rule_name": "Google Drive Ownership Transferred via Google Workspace",
     "sha256": "98600fe4b1c0c882bb99021122279f31ce5cdd2266abf34b56bab33f0cb7f190",
     "type": "query",
-    "version": 3
+    "version": 104
```

Comment:
Forked at 8.3, where max allowable is `=-` the current as a buffer. SHA256 hashes differ where the 8.3 related integrations version value is `1.0.2` and 8.4+ is `2.0.0`. Expected rule version changes should be expected for 8.4+ if the rule contents change, whereas the 8.3 fork is expected to not change unless manually.